.settings/org.eclipse.m2e.core.prefs

```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```

Comment:
Probably safe to delete this entire .settings directory

Reply:
These files keep coming back whenever I try switching branches. So I just added them to my .gitignore.